text/0265-event-vision.md

```diff
@@ -0,0 +1,77 @@
+# Event Basics
+
+## Motivation
+
+The introduction of Events has been contentious, so we want to document and agree on a few basics.
+
+### What are OpenTelemetry Events?
+
+OpenTelemetry Events are a type of OpenTelemetry Log that requires an event name and follows a specific structure implied by that event name.
+
+They are a core concept in OpenTelemetry Semantic Conventions.
+
+### OTLP
+
+Since OpenTelemetry Events are a type of OpenTelemetry Log, they share the same OTLP log data structure and pipeline.
+
+### API
+
+OpenTelemetry should provide a (user-facing) Logs API that includes the capability to emit OpenTelemetry Events.
```

Comment:
My agreement to this is conditioned on the user-facing Logs API remaining simple as it is now.

Reply:
I added this note to the open questions around Logs API capabilities and ergonomic improvements: https://github.com/open-telemetry/oteps/pull/265/commits/3c8c0abaff75a46cf58ff1aacea9e0a0092dd1a2